bin/lib/zopen-update-cacert

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+#
+# Updates the cacert in zopen
+#
+export utildir="$( cd "$(dirname "$0")" >/dev/null 2>&1 && pwd -P )"
```

Comment:
this is 'mydir' rather than 'utildir' right? would it make sense to have mydir and then create utildir as mydir/../../ ?